### PR TITLE
Update enable-kerberos.md

### PR DESCRIPTION
**Pull request by ICEPower420**
Added the correct way to enter the server details

### CODE REVIEW

**Comment by Court72** [comment]
@markingmyname

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/azure-data-studio/enable-kerberos.md [hunk 1]
```diff
@@ -161,7 +161,7 @@ krbtgt/DOMAIN.COMPANY.COM@ DOMAIN.COMPANY.COM.
 
 1. Create a new connection profile.
 
-1. Select **Windows Authentication** as the authentication type.
+1. Enter for Server hostname.DOMAIN.COMPANY.COM and select **Windows Authentication** as the authentication type.
```
Comment (rwestMSFT) [inline docs/azure-data-studio/enable-kerberos.md]:
```suggestion
1. Select **Windows Authentication** as the authentication type.

1. For the Server, enter a fully qualified host name, in the format `hostname.DOMAIN.CONTOSO.COM`.
```

**Comment by ICEPower420** [comment]
Small comment regarding step:
For the Server, enter a fully qualified host name, in the format `hostname.DOMAIN.CONTOSO.COM`

The article starts with running nltest with output of:
 `nltest /dsgetdc:DOMAIN.COMPANY.COM (where "DOMAIN.COMPANY.COM" maps to your domain's name)`

It would be confusing to the reader to talk about a fully qualified host name of domain.CONTOSO.com at this stage instead of domain.COMPANY.com.

**Comment by rwestMSFT** [comment]
@ICEPower420 That's an oversight from the original article, so I'll update that. Thanks.

**Comment by rwestMSFT** [comment]
We're going to include your change into an internal PR. When that is merged, this PR will automatically close. You can expect to see the latest article in the next day or so.

**Pull request closed without merging**